packages/embarkjs/ens/package.json

```diff
@@ -33,13 +34,15 @@
   ],
   "embark-collective": {
     "build:browser": true,
-    "build:node": true
+    "build:node": true,
+    "typecheck": true
   },
   "scripts": {
     "_build": "npm run solo -- build",
+    "_typecheck": "npm run solo -- typecheck",
     "ci": "npm run qa",
     "clean": "npm run reset",
-    "qa": "npm-run-all _build",
+    "qa": "npm-run-all _typecheck _build",
```

Comment:
If a package wasn't set up for linting prior to this PR, I didn't add a lint script/s, since that would likely involve even more changes re: fixing linter errors. I would prefer to remedy the lack of linting in some packages when I implement collective lint in a future PR.